src/main/java/persistence/meta/vo/TableName.java

```diff
@@ -0,0 +1,30 @@
+package persistence.meta.vo;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+public class TableName {
+    private final String tableName;
+    public static TableName createFromClass(Class<?> cls) {
```

Comment:
```suggestion
    private final String tableName;
    
    public static TableName createFromClass(Class<?> cls) {
```
메소드와 필드명은 한칸 공백을 주는 것이 가독성이 좋아보입니다